purchase.adoc

```diff
@@ -153,9 +247,9 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
         "accuracyMeters": 65
       },
       "purchaseNumber": 9,
-      "userDisplayName": "Pär Svanström",
-      "userId": 5210,
-      "organizationId": 6927,
+      "userDisplayName": "Elna Westman",
```

Comment:
That's actually a randomised name, created by my sandbox data loader. But I like the Hitchhiker's Guide so why not? :)